test/v1/controllers/requestParsers/DeleteOtherDeductionsRequestParserSpec.scala

```diff
@@ -41,7 +42,7 @@ class DeleteOtherDeductionsRequestParserSpec extends UnitSpec {
         MockDeleteOtherDeductionsValidator.validate(inputData).returns(Nil)
 
         parser.parseRequest(inputData) shouldBe
-          Right(DeleteOtherDeductionsRequest(Nino(nino), "2017-18"))
+          Right(DeleteOtherDeductionsRequest(Nino(nino), TaxYear.fromMtd(taxYear)))
```

Comment:
can we update the val taxYear above on line 28 to "2019-20"

Reply:
Updated